pkg/conversion/converter.go

```diff
@@ -670,6 +670,32 @@ func (c *Converter) defaultConvert(sv, dv reflect.Value, scope *scope) error {
 		switch st.Kind() {
 		case reflect.Ptr, reflect.Interface:
 			return c.convert(sv.Elem(), dv.Elem(), scope)
+
+		case reflect.Slice:
```

Comment:
I wish there was a place more specific to query param encoding/decoding to put this... code this complex in a shared codepath will never get removed :-/